app/views/documents/form_partials/_tags.html.erb

```diff
@@ -13,35 +14,69 @@
       <% end %>
     <% end %>
 
-
-    <div class="govuk-grid-row">
-      <div class="govuk-grid-column-one-half">
-        <h3 class="govuk-heading-s govuk-!-padding-top-4">
-          PLANS
-        </h3>
-        <div class="govuk-checkboxes govuk-checkboxes--small">
-          <%= form.collection_check_boxes :tags, Document::PLAN_TAGS, :itself, :itself do |b| %>
-            <div class="govuk-checkboxes__item">
-              <%= b.check_box class: "govuk-checkboxes__input" %>
-              <%= b.label class: "govuk-label govuk-checkboxes__label tag_checkbox_label" %>
+    <% @planning_application.application_type.document_tags.keys.each do |key| %>
+      <div class="govuk-grid-row">
+        <div class="govuk-grid-column-full">
+          <h3 class="govuk-heading-s">
+            <%= key.humanize %>
+          </h3>
+          <div class="govuk-grid-column-one-half">
+            <div class="govuk-checkboxes govuk-checkboxes--small">
+              <%= form.collection_check_boxes :tags, tags_first_half(@planning_application.application_type.document_tags[key]), :itself, :itself do |b| %>
+                <div class="govuk-checkboxes__item">
+                  <%= b.check_box class: "govuk-checkboxes__input" %>
+                  <%= b.label class: "govuk-label govuk-checkboxes__label tag_checkbox_label" %>
+                </div>
+              <% end %>
             </div>
-          <% end %>
-        </div>
-      </div>
-
-      <div class="govuk-grid-column-one-half">
-        <h3 class="govuk-heading-s govuk-!-padding-top-5">
-          EVIDENCE
-        </h3>
-        <div class="govuk-checkboxes govuk-checkboxes--small">
-          <%= form.collection_check_boxes :tags, Document::EVIDENCE_TAGS, :itself, :itself do |b| %>
-            <div class="govuk-checkboxes__item">
-              <%= b.check_box class: "govuk-checkboxes__input" %>
-              <%= b.label class: "govuk-label govuk-checkboxes__label tag_checkbox_label" %>
+          </div>
+          <div class="govuk-grid-column-one-half">
+            <div class="govuk-checkboxes govuk-checkboxes--small">
+              <%= form.collection_check_boxes :tags, tags_second_half(@planning_application.application_type.document_tags[key]), :itself, :itself do |b| %>
+                <div class="govuk-checkboxes__item">
+                  <%= b.check_box class: "govuk-checkboxes__input" %>
+                  <%= b.label class: "govuk-label govuk-checkboxes__label tag_checkbox_label" %>
+                </div>
+              <% end %>
```

Comment:
Sorry, wasn't clear - you can just use `in_groups` directly with a block, e.g.

``` erb
<% @planning_application.application_type.document_tags[key].in_groups(2, false) do |tags| %>
  <div class="govuk-grid-column-one-half">
    <div class="govuk-checkboxes govuk-checkboxes--small">
      <%= form.collection_check_boxes :tags, tags, :itself, :itself do |b| %>
        <div class="govuk-checkboxes__item">
          <%= b.check_box class: "govuk-checkboxes__input" %>
          <%= b.label class: "govuk-label govuk-checkboxes__label tag_checkbox_label" %>
        </div>
      <% end %>
    </div>
  </div>
<% end %>
```

You can do the same below with `irrelevant_tags`

Reply:
ahhh gotcha.